static/templates/settings/deactivation-stream-modal.handlebars

```diff
@@ -1,4 +1,4 @@
-<div id="deactivation_stream_modal" class="modal modal-bg hide fade" tabindex="-1" role="dialog" aria-labelledby="deactivation_stream_modal_label" aria-hidden="true">
+<div id="deactivation_stream_modal" class="modal modal-bg hide fade new-style" tabindex="-1" role="dialog" aria-labelledby="deactivation_stream_modal_label" aria-hidden="true">
```

Comment:
Do we have any modals that don't have `new-style` on them?  It's possible we can just migrate the world to all modals using the new-style styling as a follow-up.  (I guess I don't remember what this does for modals specifically).